testdata/unchecked-type-assertion.go

```diff
@@ -0,0 +1,58 @@
+package fixtures
```

Comment:
Can we have a test that will check this:

if foo.(int) == 1 {
    return
}

Is it supported?

Reply:
Also, consider this example:

```go
	var value any
	value = 42 // int

	if v := value.(int); v == 42 {
		fmt.Printf("Value is an integer: %d\n", v)
	}
```

Will the linter check and fail here?